src/r2b2/tests/test_audit.py

```diff
@@ -18,55 +22,64 @@ def compute_sample_size(self):
 
 
 def test_simple_audit():
```

Comment:
I agree that we will need `__str__` and `__repr__` methods for our classes. 

With respect to doctests, I was under the impression that doctests are not useful for unit testing because they are less efficient and do not maintain the setup between doctests which would make writing tests for object creation rather difficult. While it is possible to use them for unit testing, I usually only see doctests used for documenting examples of code usage, not for unit testing.

Reply:
I think this doctest would be a simple two-liner: invocation and expected result, and would replace the whole 10-line function. It would also provide an example which is easy for users to copy-paste and play with.

I agree that in general, complicated testing needs more intricate setups in unit tests.